bundles/org.openhab.binding.velbus/README.md

```diff
@@ -107,6 +107,27 @@ vmb1bl, vmb1bls, vmb1dm, vmb1led, vmb1ry, vmb1ryno, vmb1rynos, vmb1rys, vmb1ts,
 
 `[CHx="..."]` is optional, and represents the name of channel x, e.g. CH1 specifies the name of channel 1.
 
+
+The `vmbel1`, `vmbel2`, `vmbel4`, `vmbelpir`, `vmbel1`, `vmbgp1`, `vmbgp1-2`, `vmbgp2`, `vmbgp2-2`, `vmbgp4`, `vmbgp4-2`, `vmbgp4pir` and `vmbgp4pir-2` thing type also supports 1 subaddress. This subaddress is the Sensor Address. The value "FF" = not configured.
+
+```java
+Thing velbus:vmbel1:<bridgeId>:<thingId> [subaddress1="A1"]
+```
+
+The `vmbelo`, `vmbgpo`, `vmbgpod` and `vmbgpod-2` thing type also supports 4 subaddresses. The subaddress4 is the Sensor Address. The value "FF" = not configured.
+
+```java
+Thing velbus:vmbelo:<bridgeId>:<thingId> [subaddress1="B1", subaddress2="B2", subaddress3="B3", subaddress4="B4"]
+```
+
+The `vmbdali` thing also supports 9 subaddresses. The value "FF" = not configured.
+
+```java
+Thing velbus:vmbelo:<bridgeId>:<thingId> [subaddress1="F1", subaddress2="F2", subaddress3="F3", subaddress4="F4", subaddress5="F5", subaddress6="F6", subaddress7="F7", subaddress8="F8", subaddress9="F9"]
+```
+
+In the UI, the subaddress settings are in the advanced configuration of the module.
```

Comment:
Is this still true? The `config.xml` does not contain `<advanced>true</advanced>` for the subaddress configuration.

Reply:
Whoops, I forgot to remove this during my last changes.

Reply:
Updated in last commit.